applications/RomApplication/custom_utilities/rom_residuals_utility.h

```diff
@@ -138,17 +140,183 @@ namespace Kratos
         return MatrixResiduals;
         }
 
+        Matrix GetProjectedResidualsOntoPsi()
+        {
+            // Getting the number of elements and conditions from the model
+            const int nelements = static_cast<int>(mpModelPart.Elements().size());
+            const int nconditions = static_cast<int>(mpModelPart.Conditions().size());
+
+            const auto& CurrentProcessInfo = mpModelPart.GetProcessInfo();
+            const auto el_begin = mpModelPart.ElementsBegin();
+            const auto cond_begin = mpModelPart.ConditionsBegin();
+
+            //contributions to the system
+            Matrix LHS_Contribution = ZeroMatrix(0, 0);
+            Vector RHS_Contribution = ZeroVector(0);
+
+            //vector containing the localization in the system of the different terms
+            Element::EquationIdVectorType EquationId;
+            Matrix MatrixResiduals( (nelements + nconditions), mPetrovGalerkinRomDofs); // Matrix of reduced residuals.
+            Matrix PsiElemental;
```

Comment:
```suggestion
            Element::EquationIdVectorType equation_id;
            Matrix matrix_residuals( (nelements + nconditions), mPetrovGalerkinRomDofs); // Matrix of reduced residuals.
            Matrix psi_elemental;
```
Note the `snake_case` convention for local variables.